adflow/pyADflow.py

```diff
@@ -4599,34 +4552,34 @@ def _getDefOptions(self):
             'cflcoarse':[float, 1.0],
             'mgcycle':[str, '3w'],
             'mgstartlevel':[int, -1],
-            'resaveraging':[str,'alternateresaveraging'],
+            'resaveraging':[str, ['alternateresaveraging', 'noresaveraging', 'alwaysresaveraging']],
```

Comment:
Are you planning to update these names?

Reply:
I decided not to for now to keep the backward incompatible changes to a minimum. I could do it though if you think it's worth changing.

Reply:
I personally think if we are to make backwards incompatible changes we might as well do them at once. That's why I also asked about #41, I think it's more work to announce/release changes multiple times rather than all at once. Up to you guys though since I'm not even a maintainer for adflow.

Reply:
Good point about "squashing" the backwards incompatible changes into one commit. 

I was still in the mindset of 1 change per PR. 
Is there a way to still do one PR per feature and only make one update on the master branch?
Could we collect some backwards incompatible changes in a separate branch?

Reply:
I was not thinking about one commit vs. multiple on the master branch, IMO it's fine to have multiple backwards incompatible changes on the master branch. It's more a matter of when to do the releases since my mindset is that that's when changes are formalized, listed and communicated to end users. It's generally not a good idea to just pull down the latest changes on master, and checking out a tagged release should help remind users to read the release notes.

Reply:
How would you all feel about making these changes, but instead of causing an error, incorrect capitalization leads to a deprecation warning. 

Then we can Merge this PR but avoid incrementing the major version number.